af/fastpath/fastpath/core.py

```diff
@@ -1429,6 +1424,25 @@ def score_riseupvpn(msm: dict) -> dict:
     return scores
 
 
+def score_openvpn(msm: dict) -> dict:
+    scores = init_scores()
+    tk = g_or(msm, "test_keys", {})
+    tstatus = tk.get("transport_status") or {}
+    obfs4 = tstatus.get("obfs4")
+    openvpn = tstatus.get("openvpn")
+    anomaly = (
+        tk.get("api_status") == "blocked"
+        or tk.get("ca_cert_status") is False
+        or obfs4 == "blocked"
+        or openvpn == "blocked"
+    )
+    if anomaly:
+        scores["blocking_general"] = 1.0
+
+    scores["extra"] = dict(test_runtime=msm.get("test_runtime"))
+    return scores
```

Comment:
On what test specification and implementation is this parsing/scoring logic based on?

I checked the spec for the OpenVPN test (https://github.com/ooni/spec/pull/255) and the code for it (https://github.com/ooni/probe-cli/blob/8e26e4993bf48b90a1a89e8a6ada69d6e42b728e/internal/engine/experiment/openvpn/openvpn.go) in the relevant PR: https://github.com/ooni/probe-cli/pull/845, but could not find reference to the keys `transport_status` or `api_status`.

Reply:
This is based on Ain's feedback and the unit test files provided.

Reply:
Initial basic scoring added.